python/pyspark/sql/tests/test_arrow_python_udf.py

```diff
@@ -219,6 +220,17 @@ def test_udf(a, b):
         with self.assertRaises(PythonException):
             self.spark.sql("SELECT test_udf(id, a => id * 10) FROM range(2)").show()
 
+    def test_udt_as_return_type(self):
```

Comment:
Shall we add a test for Python UDTF as well (in `test_udtf`)?

Reply:
DOne